recipes/tests.rb

```diff
@@ -183,6 +186,7 @@
   bash 'test CUDA install' do
     cwd Chef::Config[:file_cache_path]
     code <<-TESTCUDA
+      set -e
```

Comment:
must be not put here, it's few lines later

Reply:
OK :+1: